health-services/ExerciseSampleCompose/app/src/main/java/com/example/exercisesamplecompose/service/ExerciseState.kt

```diff
@@ -11,6 +12,7 @@ data class ExerciseMetrics(
     val distance: Double? = null,
     val calories: Double? = null,
     val heartRateAverage: Double? = null,
+    val exerciseGoal: Boolean = false
```

Comment:
Instead of a boolean, should the goals be here?

Reply:
I decided to make this a boolean because what the Exercise Update returns is a set of goals that were achieved, for example: 
`[ExerciseGoal(exerciseGoalType=ONE_TIME_GOAL, dataTypeCondition=DataTypeCondition(dataType=DataType(name=Active Exercise Duration, timeType=INTERVAL, class=Long, isAggregate=true), threshold=30, comparisonType=GREATER_THAN_OR_EQUAL), period=null)]`

For the purposes of the sample, in order to trigger the confirmation dialog, I'm just interested in whether or not a goal is returned at all. But I do see how it might make more sense to have the goals returned, then check to see if the set is empty later on.

Reply:
I've adjusted it to return goals then check to see if the set is empty in the new update. That way if someone is referencing the sample, they can use that infrastructure, then decide how to interpret it

Reply:
I also realized I had defined exerciseGoals twice -- this was was not needed as it shouldn't be considered an Exercise metric